### PR TITLE
Updated package-lock.json for the lodash patch

### PR DESCRIPTION
**Pull request by binarymist**
Just needs publishing to NPM now @kingthorin

### CODE REVIEW

**Comment by binarymist** [comment]
Usually best to upgrade via `npm install`. package-lock.json is a generated file, shouldn't be changed by hand.

**Comment by thc202** [comment]
Mind update the changelog?

**Comment by thc202** [comment]
The version of the package should be updated as well?

**Comment by binarymist** [comment]
I should give this a bit of a test before you publish. May not get a chance until next weekend.
> Mind update the changelog?
Will do.
> The version of the package should be updated as well?
Will do.

**Comment by kingthorin** [comment]
> I should give this a bit of a test before you publish. May not get a chance until next weekend.

No worries, just drop an update here once you get a chance

**Comment by binarymist** [comment]
[20 July]

I'm bogged down with other work currently

**Comment by binarymist** [comment]
@thc202 All yours.

**Comment by binarymist** [comment]
Hay @thc202 were you going to publish this?

**Comment by thc202** [comment]
Sorry, missed your comments. I'll check today.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
```
Comment (thc202) [inline CHANGELOG.md]: This line should be removed, per new version added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
+## [1.0.1] - 2019-08-03 - lodash security fix
```
Comment (thc202) [inline CHANGELOG.md]: The date needs to be updated.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+### Security
+
+* Regenerated the `package-lock.json` to capture the fixed version of sub dependency lodash (4.17.11 -> 4.17.14)
```
Comment (thc202) [inline CHANGELOG.md]: Does it worth update again? There's a new version of lodash.

File: CHANGELOG.md [hunk 4]
```diff
@@ -33,5 +38,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## 0.3.0 - 2017-12-04
 
 
-[Unreleased]: https://github.com/zaproxy/zap-api-nodejs/compare/v1.0.0-rc.1...HEAD
+[1.0.1]: https://github.com/zaproxy/zap-api-nodejs/compare/v1.0.0-rc.1...zaproxy:v1.0.1
+
+[Unreleased]: https://github.com/zaproxy/zap-api-nodejs/compare/v1.0.1...HEAD
```
Comment (thc202) [inline CHANGELOG.md]: This should be removed as well (and the empty line above).

File: CHANGELOG.md [hunk 5]
```diff
@@ -33,5 +38,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## 0.3.0 - 2017-12-04
 
 
-[Unreleased]: https://github.com/zaproxy/zap-api-nodejs/compare/v1.0.0-rc.1...HEAD
+[1.0.1]: https://github.com/zaproxy/zap-api-nodejs/compare/v1.0.0-rc.1...zaproxy:v1.0.1
```
Comment (thc202) [inline CHANGELOG.md]: The `zaproxy:` prefix can be removed.

**Comment by thc202** [comment]
Let me know if you can tackle the comments shortly, otherwise I can do that.

**Comment by thc202** [comment]
Addressed the comments.

**Comment by thc202** [comment]
For the record, version 1.0.1 was already published to NPM.

Thanks @binarymist!

**Comment by binarymist** [comment]
Anyone (@kingthorin, @psiinon)?

**Comment by binarymist** [comment]
Sorry, never saw your comments until now @thc202

**Comment by thc202** [comment]
No problem.

**Pull request merged**